zookeeper-server/src/main/java/org/apache/zookeeper/cli/GetCommand.java

```diff
@@ -38,10 +37,12 @@ public class GetCommand extends CliCommand {
     static {
         options.addOption("s", false, "stats");
         options.addOption("w", false, "watch");
+        options.addOption("b", false, "base64");
+        options.addOption("x", false, "hexdump");
     }
 
     public GetCommand() {
-        super("get", "[-s] [-w] path");
+        super("get", "[-s] [-w] [-b] [-x] path");
```

Comment:
single character flags that have no description aren't very useful. -b64 and -hex would be easier to immediately understand

Reply:
> Single character flags that have no description aren't very useful.

Yes, I agree. The current Zookeeper CLI command usage isn't very appropriate. Maybe we could improve it in another PR.

> -b64 and -hex would be easier to immediately understand

I kept the short option naming convention here and just improved the `get` and `set` command usage with additional option descriptions. Here is the `get` command usage after the change:
```
get [-s] [-w] [-b] [-x] path
 -b   Output data in base64 format
 -s   Print znode stats additionally
 -w   Watch for changes on the znode
 -x   Output data in hexdump format

```